modules/library/referencing/src/main/java/org/geotools/measure/SexagesimalConverter.java

```diff
@@ -118,6 +166,12 @@ public int hashCode() {
         return (int) serialVersionUID + divider;
     }
 
+    @Override
+    protected String transformationLiteral() {
+        // TODO
+        return null;
```

Comment:
While `null` is valid for the api, is there anything useful to return here? If not we can remove the todo.

Reply:
`	 * Returns a String describing the transformation that is represented by this converter. 
	 * Contributes to converter's {@code toString} method. If null or empty
	 * {@code toString} output becomes simplified.`

When you toString the converter it will call this method and append the value to the end of the to string if it is not null.  I couldn't think if we should add anything there or not.

Reply:
Yes, I checked the base class before commenting, ... so we can remove the `// TODO` ?

Reply:
Removed TODO statement.